pkg/ipcache/kvstore.go

```diff
@@ -217,8 +220,10 @@ restart:
 				goto restart
 			}
 
-			scopedLog := log.WithFields(logrus.Fields{"kvstore-event": event.Typ.String(), "key": event.Key})
-			scopedLog.Debug("Received event")
+			if option.Config.Debug {
+				scopedLog = log.WithFields(logrus.Fields{"kvstore-event": event.Typ.String(), "key": event.Key})
```

Comment:
AFAICT `scopedLog` is never used again - why not just do `log.WithFields(logrus.Fields{"kvstore-event": event.Typ.String(), "key": event.Key}).Debug("Received event")` ?

Reply:
Looks like it is used again, in the `nil` IP create/modify case below.